gitlab/golang/stages/50-deployment/debrelease.yaml

```diff
@@ -0,0 +1,14 @@
+variables:
+  DEB_BINARY: "${CI_PROJECT_NAME}_${CI_COMMIT_TAG}_linux_amd64.deb"
+  PACKAGE_REGISTRY_URL: "${CI_API_V4_URL}/projects/${CI_PROJECT_ID}/packages/generic/${CI_PROJECT_NAME}/${CI_COMMIT_TAG}"
+
+delivery:debrelease:
```

Comment:
This should be in the folder delivery right?

Reply:
Oh it's supposed to be in deployment. I'll change the name

Reply:
We do need the build and release to be in two steps.